libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.html

```diff
@@ -1,64 +1,49 @@
-<button
-  class="sky-help-inline"
-  type="button"
-  [attr.aria-controls]="
-    ariaControls
-      | skyHelpInlineAriaControls
-        : popoverId
-        : helpKey
-        : (helpSvc?.widgetReadyStateChange | async)
-  "
-  [attr.aria-expanded]="
-    ariaExpanded | skyHelpInlineAriaExpanded: ariaControls : isPopoverOpened
-  "
-  [attr.aria-haspopup]="helpKey | skyHelpInlineAriaHaspopup"
-  [attr.aria-label]="
-    labelledBy
-      ? undefined
-      : labelText
-        ? ('skyux_help_inline_aria_label' | skyLibResources: labelText)
-        : ariaLabel
-          ? ariaLabel
-          : ('skyux_help_inline_button_title' | skyLibResources)
-  "
-  [attr.aria-labelledby]="
-    !labelledBy ? undefined : labelTextElement.id + ' ' + labelledBy
-  "
-  [ngClass]="{
-    'sky-help-inline-hidden': helpKey && !helpSvc && !popoverContent
-  }"
-  [skyPopover]="popoverContent ? popoverRef : undefined"
-  (click)="onClick()"
->
-  <sky-icon *skyThemeIf="'default'" icon="info-circle" />
-  <sky-icon-stack
-    *skyThemeIf="'modern'"
-    size="xs"
-    [baseIcon]="{
-      icon: 'circle-solid',
-      iconType: 'skyux'
-    }"
-    [topIcon]="{
-      icon: 'help-i',
-      iconType: 'skyux'
-    }"
-  />
-</button>
+@let ariaLabelResolved =
+  ariaLabel
+    | skyHelpInlineAriaLabel
+      : labelTextResolved()
+      : labelledBy
+      : defaultAriaLabel();
```

Comment:
`@let` works in Angular 18. When we backport this to `10.x.x` I'll just copy/paste this assignment to all three `ariaLabel` inputs below.